src/DssSpell.t.sol

```diff
@@ -946,11 +950,11 @@ contract DssSpellTest is DSTest, DSMath {
             calc_cut:     9950
         });
         afterSpell.collaterals["PSM-USDC-A"] = CollateralValues({
```

Comment:
Why test of PSM-USDC-A is modified? There are no related updates in the spell itself?

Reply:
https://vote.makerdao.com/polling/QmZz4ssm?network=mainnet#poll-detail

See line 80 in the spell.

Reply:
Now I see it.